gradle.properties

```diff
@@ -26,4 +26,8 @@ POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=amazonwebservices
 POM_DEVELOPER_ORGANIZATION=Amazon Web Services
-POM_DEVELOPER_ORGANIZATION_URL=http://aws.amazon.com
\ No newline at end of file
+POM_DEVELOPER_ORGANIZATION_URL=http://aws.amazon.com
+
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
If no dependencies use support-era dependencies — I suggest to remove this line. No need to run Jetifier on artifacts if not necessary.

Reply:
> If no dependencies use support-era dependencies — I suggest to remove this line. No need to run Jetifier on artifacts if not necessary.

Thanks for the callout. After this #1990 goes in, I'm going to come back through and update a bunch of dependencies, to their latest stable release versions. I will re-evaluate the need for `android.enableJetifier=true` once I have completed the dependencies updates.